components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/MessageProcessorResource.java

```diff
@@ -117,19 +126,45 @@ public boolean invoke(MessageContext messageContext) {
         return true;
     }
 
+    private static List<MessageProcessor> getSearchResults(MessageContext messageContext, String searchKey){
```

Comment:
Fix formatting. There should be a space before {. Please fix in other places too.